src/lambda/vue/samInvokeVue.ts

```diff
@@ -298,6 +298,18 @@ export const Component = Vue.extend({
                     },
                 },
             })
+        },
+        clearForm() {
+            this.launchConfig = newLaunchConfig()
+            this.containerBuildStr = ''
+            this.skipNewImageCheckStr = ''
+            this.payload = {value: '', errorMsg: ''}
```

Comment:
I guess this means we are not use a "data binding" concept for this form, correct? Else I would expect `this.launchConfig = newLaunchConfig()` to clear the form implicitly.

Reply:
Properties that are arrays or objects could not be directly bound. Properties of string and number currently are bound.